src/parsing.jl

```diff
@@ -331,6 +362,10 @@ function Base.parse(::Type{ZonedDateTime}, pqv::PQValue{PQ_SYSTEM_TYPES[:int8]})
     return TimeZones.unix2zdt(parse(Int64, pqv))
 end
 
+function Base.parse(::Type{UTCDateTime}, pqv::PQValue{PQ_SYSTEM_TYPES[:int8]})
+    return UTCDateTime(unix2datetime(parse(Int64, pqv)))
```

Comment:
Since this PR is closed I can no longer make suggestions, but

```
UTCDateTime(unix2datetime(parse(Int64, pqv)))
```
can be 
```
UTCDateTime(parse(DateTime, pqv))
```